### PR TITLE
python3 support

### PR DESCRIPTION
**Pull request by crvcrv**

### CODE REVIEW

**Comment by untitaker** [comment]
Uhm.

**Comment by methane** [comment]
Why this change helps Python 3 support?

**Comment by crvcrv** [comment]
sorry I did some late night work and I shouldn't've committed at that time without double checking first. I'm going to fix this commit later on.

**Comment by crvcrv** [comment]
fixed my previous commit. 
ran tests with 2.7 and 3.3 and both pass

File: flask_cache/jinja2ext.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```
Comment (untitaker) [inline flask_cache/jinja2ext.py:2]: I think these changes are not useful at all.

**Comment by thadeusb** [comment]
callable is allowed in python 3.3

**Pull request closed without merging**